release/releaser.go

```diff
@@ -261,8 +261,44 @@ func selectServices(rc *ReleaseContext, spec *flux.ReleaseSpec, results flux.Rel
 	}
 
 	// Build list of filters
-	var filtList []ServiceFilter
+	filtList, err := filters(spec, conf)
+	if err != nil {
+		return nil, err
+	}
+
+	// For backwards-compatibility, there's two fields: ServiceSpec
+	// and ServiceSpecs. An entry in ServiceSpec takes precedence.
+	switch spec.ServiceSpec {
```

Comment:
It might make sense to do this as part of the `filters` function, so all the service spec -> filter conversion happens in one place.